.coveragerc

```diff
@@ -2,11 +2,12 @@
 omit =
     */python?.?/*
     */site-packages/nose/*
+    /cltk/tests/*    
 exclude_lines =
     pragma: no cover
     def __repr__
     if self\.debug
     if 0:
     if __name__ == .__main__.:
 #    raise AssertionError
```

Comment:
Looked at the history. Patrick, you added tis extra stuff, which looks fine. I believe we should delete a couple, though: both the errors, and `if 0:`.

Reply:
Hmmm—definitely added /cltk/tests/* (not set on it if you'd prefer) because of decreased coverage; seemed oddly circular to me for the tests to be decreasing coverage, maybe I'm missing something. Don't remember adding the others, and definitely not recently. Let me look at the history and get back to you.

Reply:
That's logical. I thought nose would handle `test_` objects smartly. Not a big deal, but we should know the answer.